docs/framework/windows-workflow-foundation/toc.yml

```diff
@@ -9,6 +9,9 @@
     href: whats-new-in-wf-in-dotnet.md
   - name: Deprecated types in Windows Workflow Foundation
     href: deprecated-types.md
+    items:
+    - name: "wfc.exe Workflow Compiler Tool"
```

Comment:
Did you mean to make this a subnode of "Deprecated types"?

Reply:
yes, wfc.exe is listed as deprecated in the parent link (https://docs.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/deprecated-types).